tt_metal/host_api.hpp

```diff
@@ -283,7 +283,7 @@ std::unique_ptr<GlobalSemaphore> CreateGlobalSemaphore(
     Device *device, CoreRangeSet &&cores, uint32_t initial_value, BufferType buffer_type = BufferType::L1);
 
 /**
-*  Creates a pre-allocated interleaved DRAM or L1 buffer with the global allocator on device
+*  Creates a pre-allocated interleaved DRAM or L1 buffer with the global allocator on the device. Interleaved buffers allow for frequent attribute access, improving performance by storing data locally. For more information about buffers visit the Tenstorrent GitHub: [Buffers info on GitHub](https://github.com/tenstorrent/tt-metal/tree/main/tt_metal/impl/buffers).
```

Comment:
"storing data locally" is actually sharded buffers. 

`frequent attribute access`: What does this mean?